src/main/java/core/basesyntax/service/Convert.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface Convert {
+    List<FruitTransaction> convertToJavaObject(List<String> dataFileList);

```

Comment:
you have determined return type, so it's understandable what you are converting to
```suggestion
    List<FruitTransaction> parseAll(List<String> dataFileList);

```